simulator/client/client.mjs

```diff
@@ -14,6 +14,22 @@ const waitForPort = async () => {
   console.log(r)
 }
 
+class HTTPResponseError extends Error {
+	constructor(response, ...args) {
```

Comment:
I'm just noticing that there are some heretical indentations here. We want 2 spaces